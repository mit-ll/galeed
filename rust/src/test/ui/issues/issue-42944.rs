mod foo {
    pub struct Bx(());
}

mod bar {
    use foo::Bx;

    fn foo() {
        Bx(());
        //~^ ERROR expected function, tuple struct or tuple variant, found struct `Bx` [E0423]
    }
}

mod baz {
    fn foo() {
        Bx(());
        //~^ ERROR cannot find function, tuple struct or tuple variant `Bx` in this scope [E0425]
    }
}

fn main() {}
