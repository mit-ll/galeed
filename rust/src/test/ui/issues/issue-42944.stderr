error[E0423]: expected function, tuple struct or tuple variant, found struct `Bx`
  --> $DIR/issue-42944.rs:9:9
   |
LL |         Bx(());
   |         ^^ constructor is not visible here due to private fields

error[E0425]: cannot find function, tuple struct or tuple variant `Bx` in this scope
  --> $DIR/issue-42944.rs:16:9
   |
LL |         Bx(());
   |         ^^ not found in this scope
   |
help: consider importing this tuple struct
   |
LL |     use foo::Bx;
   |

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0423, E0425.
For more information about an error, try `rustc --explain E0423`.
